src/youtube/js/youtube.js

```diff
@@ -32,7 +32,7 @@ require("../../../html5-common/js/utils/constants.js");
     this.features = [ OO.VIDEO.FEATURE.VIDEO_OBJECT_SHARING_GIVE,
                       OO.VIDEO.FEATURE.BITRATE_CONTROL ];
     this.technology = OO.VIDEO.TECHNOLOGY.HTML5;
-    this.encodings = [OO.VIDEO.ENCODING.MP4];
+    this.encodings = [OO.VIDEO.ENCODING.YOUTUBE];
```

Comment:
Shoudl there only be 1 entry? Do we need both Youtube and MP4?  Just double checking.